npm/ng-packs/packages/theme-shared/extensions/src/lib/components/extensible-form/extensible-form-prop.component.html

```diff
@@ -7,7 +7,7 @@
     </ng-container>
   </ng-template>
 
-  <div [ngClass]="containerClassName" class="mb-3">
+  <div [ngClass]="containerClassName" class="mb-2">
```

Comment:
why we have two times

Reply:
default value
![image](https://github.com/abpframework/abp/assets/72804437/fecb276c-3e03-4bad-8283-c4e50648009b)

if FormProp has className attribute
![image](https://github.com/abpframework/abp/assets/72804437/aacff425-c09c-4edf-a57e-7c086e62be7a)

then it can be used